src/slurmjobs.rs

```diff
@@ -415,77 +437,83 @@ fn push_uint(obj: &mut output::Object, name: &str, val: &str) {
     push_uint_full(obj, name, val, 1, 0, false);
 }
 
-fn push_uint_full(obj: &mut output::Object, name: &str, val: &str, scale: u64, bias: u64, always: bool) {
+fn push_uint_full(
+    obj: &mut output::Object,
+    name: &str,
+    val: &str,
+    scale: u64,
+    bias: u64,
+    always: bool,
+) {
     if val != "" {
         match val.parse::<u64>() {
-            Ok(n) => if n != 0 || bias != 0 || always {
-                obj.push_u(name, bias + n*scale);
+            Ok(n) => {
+                if n != 0 || bias != 0 || always {
+                    obj.push_u(name, bias + n * scale);
+                }
             }
-            Err(_) => { }
+            Err(_) => {}
         }
     }
 }
 
 fn push_duration(obj: &mut output::Object, name: &str, mut val: &str) {
     // [DD-[hh:]]mm:ss
-    let days =
-        if let Some((dd,rest)) = val.split_once('-') {
-            if let Ok(n) = dd.parse::<u64>() {
-                val = rest;
-                n
-            } else {
-                return;
-            }
-        } else {
-            0
-        };
-    let mut elts = val.split(':').collect::<Vec<&str>>();
-    let hours =
-        if elts.len() == 3 {
-            if let Ok(n) = elts[0].parse::<u64>() {
-                elts.remove(0);
-                n
-            } else {
-                return;
-            }
-        } else {
-            0
-        };
-    let minutes =
-        if let Ok(n) = elts[0].parse::<u64>() {
+    let days = if let Some((dd, rest)) = val.split_once('-') {
```

Comment:
Obviously I'm not going to argue with fmt, so this is just me airing a grievance, but this type of decision is just fundamentally wrong, since the nested if will appear at lower nesting levels (visually) than the outer if.